components/scream/src/dynamics/homme/atmosphere_dynamics.hpp

```diff
@@ -45,6 +45,9 @@ class HommeDynamics : public AtmosphereProcess
   // Note: field_mgrs[grid_name] is the FM on grid $grid_name
   void register_fields (const std::map<std::string,std::shared_ptr<FieldManager<Real>>>& field_mgrs) const;
 
+  // Retrieves an internal field, given field name and grid name.
+  const Field<Real>& get_internal_field (const std::string& name, const std::string& grid) const;
```

Comment:
To do the `phis` check in `homme_standalone.cpp`, I need to access an internal field, and it needed to be const. So I've changed it a bit. Is this ok?

Reply:
Yeah, that's ok. I made it non-const originally, cause it was all used inside the class only, but this does not create issues.

(Btw: I'm changing this stuff in the restart branch, to make it more structured, so no constraint right here).